src/algebra/lie/basic.lean

```diff
@@ -194,20 +197,27 @@ instance : has_one (L₁ →ₗ⁅R⁆ L₁) := ⟨{ map_lie := by simp, ..(1 :
 
 instance : inhabited (L₁ →ₗ⁅R⁆ L₂) := ⟨0⟩
 
+theorem morphism.coe_injective : function.injective (λ f : L₁ →ₗ⁅R⁆ L₂, show L₁ → L₂, from f) :=
+by rintro ⟨f, _⟩ ⟨g, _⟩ ⟨h⟩; congr
```

Comment:
```suggestion
by { rintro ⟨f, _⟩ ⟨g, _⟩ ⟨h⟩, congr, }
```
I prefer to use `;` only when there are multiple goals to discharge. Again, feel free to ignore.

Reply:
I think I'll probably leave this one for consistency with the spelling in `linear_map`, which I'd prefer not to change in order to avoid increasing the build time of this PR dramatically.